composer.json

```diff
@@ -67,10 +67,7 @@
       "web/app/themes/{$name}/": ["type:wordpress-theme"]
     },
     "wordpress-install-dir": "web/wp",
-    "drupal-scaffold": {
-      "allowed-packages": [
-        "pantheon-systems/wordpress-bedrock-integrations"
```

Comment:
@kporras07 Do we still need to define which packages are allowed to use the scaffold?

Reply:
Thank you for clarifying! That makes sense to me.